### DISCUSSION TITLE
NPM dependencies declared as "*" doesn't seem to be detected

### DISCUSSION THREAD

**Discussion by aisamu** [renovatebot/renovate#17908] [Request Help] [answered]
Is there a way to make Renovate detect (and thus pin/upgrade/etc) dependencies declared as `*`?

On a project with the following on the `package.json` (excerpt)
<img width="341" alt="Screen Shot 2022-09-20 at 10 58 25" src="https://user-images.githubusercontent.com/431708/191543425-8a3fdcad-510a-4631-b73b-dc4909d69a5e.png">

Renovate's Dashboard only lists the non-* dependencies:
<img width="368" alt="Screen Shot 2022-09-20 at 10 58 38" src="https://user-images.githubusercontent.com/431708/191544217-1762896c-507a-4e62-b5be-aa010cb3c3e5.png">

NB: Renovate works fine on the same project for non-* dependencies, detecting, pinning and/or updating them as needed.

**Comment by rarkins** [comment]
I think it might need some small code changes. Please submit a feature request with reproduction repo, including lock file if that's important to your use case.

**Comment by aisamu** [comment, marked as answer]
Issue opened: https://github.com/renovatebot/renovate/issues/17916
